16Ianuarie/src/Problem1/Problem1.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Class Problem1:
+ * -Method getMax() with two integer (int) parameters, returns maximal of the two numbers;
+ * -Three numbers are read from the console and the biggest of them will be printed;
+ * -Method getMax() is used for comparing only the first 2 numbers; the result is afterwards
+ * compared with the third number.
+ */
+
+package Problem1;
+
+import java.util.Scanner;
+
+public class Problem1 {
+
+    public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
```

Comment:
it's nice to say what you're expecting from the keyboard. System.out.println("please input the 3 numbers"):